src/flask_container_scaffold/base.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+
+"""
+Common base class to allow us to always set a code for the resource to use
+for making decisions, and to have a common place to put errors and messages.
+"""
+
+
+class BaseApiModel(BaseModel):
+    code: int = 200
```

Comment:
(question) Do we really need to include the HTTP return status code in the message? I can see this being ignored or not used properly by services and leading to confusion.

Reply:
Yes, the structure I have been using in my own code for this is that you (the caller of the api) would receive back some object that inherits from BaseApiModel, so, while perhaps slightly redundant, the object.code can be used by a client to make a decision how to process the returned data. It is also the base for input models, in which case, yes, success/fail (0/1, etc) work just fine as well. Really, it is mean to be used in both directions (in/out) for decision-making. The actual value matter less to me, HTTP code just seemed like a reasonable fit when I started using this. 
And yes, once we agree on direction, I can provide an example in the README if that would be helpful.

Reply:
Resolving this thread now that we have pulled out the 'code' parameter.